app/controllers/owners/restaurants_controller.rb

```diff
@@ -0,0 +1,12 @@
+class Owners::RestaurantsController < ApplicationController
+
+  def index
+    @owner = current_user.id
```

Comment:
We made updates to the code when doing the review of Thomas comments. Hopefully this solution will work for your comments as well.